website/docs/docs/dbt-versions/compatible-track-changelog.md

```diff
@@ -47,7 +97,7 @@ dbt-trino==1.9.0
 
 Changelogs:
 - [dbt-core 1.9.1](https://github.com/dbt-labs/dbt-core/blob/1.9.latest/CHANGELOG.md#dbt-core-191---december-16-2024)
-- [dbt-adapters 1.13.1](https://github.com/dbt-labs/dbt-adapters/blob/main/CHANGELOG.md#dbt-adapters-1131---january-10-2025)
+- [dbt-adapters 1.13.1](https://github.com/dbt-labs/dbt-adapters/blob/main/dbt-adapters/CHANGELOG.md#dbt-adapters-1131---january-10-2025)
```

Comment:
changelogs for dbt-adapters were moved as a result of the monorepo project.  This (and below) just fixes the currently broken links.